services/graphql/src/app.js

```diff
@@ -6,6 +6,7 @@ const Auth = require('./auth');
 const bearer = require('./auth/bearer');
 const env = require('./env');
 const graphql = require('./graphql/server');
+const reportingService = require('./routes/reporting');
```

Comment:
This isn't the reporting service... it's the reporting route. :) Rename to `reporting`